invokeai/app/services/processor.py

```diff
@@ -166,6 +166,15 @@ def __process(self, stop_event: Event):
                             error_type=e.__class__.__name__,
                             error=traceback.format_exc()
                         )
+                elif queue_item.invoke_all and sum(graph_execution_state.batch_indices) > 0:
```

Comment:
TODO: Switch to a boolean on the session for whether or not to continue to other sessions in the batch, or to just run the single session with the batch index provided